e2e/tests/omnibus-base-goerli-andromeda.toml/Perps_Trading.e2e.js

```diff
@@ -297,19 +297,23 @@ describe(require('path').basename(__filename, '.e2e.js'), function () {
     const marketId = 200;
     const settlementStrategyId = extras.btc_pyth_settlement_strategy;
     const maxSize = await PerpsMarketProxy.getMaxMarketSize(marketId);
+    log({ marketId, maxSize });
     try {
       await commitPerpsOrder({
         wallet,
         accountId,
         marketId,
-        sizeDelta: parseFloat(formatEther(maxSize)),
+        sizeDelta: parseFloat(ethers.utils.formatEther(maxSize.mul(2))),
```

Comment:
Isn't it better to test that the correct max is being enforced?

Reply:
because using max does not fail. max + infinitely small value does.

Reply:
🤔  How did it pass before?

Reply:
no idea, it was failing on mainnet consistently